common/main/yo.xml

```diff
@@ -716,7 +716,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		</codePatterns>
 	</localeDisplayNames>
 	<characters>
-		<exemplarCharacters>[a á à b d e é è ẹ {ẹ\u0301} {ẹ\u0300} f g {gb} h i í ì j k l m n ń o ó ò ọ {ọ\u0301} {ọ\u0300} p r s ṣ t u ú ù w y]</exemplarCharacters>
+		<exemplarCharacters>[a á à b d e é è ẹ {ẹ\u0301} {ẹ\u0300} f g {gb} h i í ì j k l m ḿ m̀ n ń ǹ o ó ò ọ {ọ\u0301} {ọ\u0300} p r s ṣ t u ú ù w y]</exemplarCharacters>
```

Comment:
```suggestion
		<exemplarCharacters>[a á à b d e é è ẹ {ẹ\u0301} {ẹ\u0300} f g {gb} h i í ì j k l m ḿ {m\u0300} n ń ǹ o ó ò ọ {ọ\u0301} {ọ\u0300} p r s ṣ t u ú ù w y]</exemplarCharacters>
```

So because `m̀` is not a single Unicode code point, it has to be enclosed in curly braces like `{m̀}` but better yet split out in hex like `{m\u0300}`. 

 ( `{m̀}` would be OK but a later BRS process would change it to the \u form)

Reply:
Educational, thanks. Two questions:
What's the advantage of splitting it out in hex? 
How will it appear in survey tool?

Reply:
I checked in the survey tool, and the accent is in hex e.g. ọ̀ appears as {ọ\u0301} in the survey tool same as in the yo.xml file.

I'm also curious what the advantage is to split it out in hex.